src/CMakeLists.txt

```diff
@@ -111,68 +120,131 @@ set(every_platform_SOURCES
 set(solvespace_core_gl_SOURCES
     solvespace.cpp)
 
-add_library(solvespace-core STATIC
-    dsc.h
-    expr.h
-    polygon.h
-    sketch.h
-    solvespace.h
-    ui.h
-    platform/platform.h
-    render/render.h
-    render/gl3shader.h
-    srf/surface.h
-    bsp.cpp
-    clipboard.cpp
-    confscreen.cpp
-    constraint.cpp
-    constrainteq.cpp
-    describescreen.cpp
-    draw.cpp
-    drawconstraint.cpp
-    drawentity.cpp
-    entity.cpp
-    export.cpp
-    exportstep.cpp
-    exportvector.cpp
-    expr.cpp
-    file.cpp
-    generate.cpp
-    graphicswin.cpp
-    group.cpp
-    groupmesh.cpp
-    importdxf.cpp
-    importidf.cpp
-    importmesh.cpp
-    mesh.cpp
-    modify.cpp
-    mouse.cpp
-    polyline.cpp
-    polygon.cpp
-    resource.cpp
-    request.cpp
-    style.cpp
-    system.cpp
-    textscreens.cpp
-    textwin.cpp
-    toolbar.cpp
-    ttf.cpp
-    undoredo.cpp
-    util.cpp
-    view.cpp
-    platform/platform.cpp
-    platform/gui.cpp
-    render/render.cpp
-    render/render2d.cpp
-    srf/boolean.cpp
-    srf/curve.cpp
-    srf/merge.cpp
-    srf/ratpoly.cpp
-    srf/raycast.cpp
-    srf/shell.cpp
-    srf/surface.cpp
-    srf/surfinter.cpp
-    srf/triangulate.cpp)
+if (NOT USE_QT_GUI)
```

Comment:
`solvespace-core` is cross platform.
If this `if` was actually needed the reason it was needed should be removed.